extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -146,9 +162,47 @@ export class AzureAccountProviderService implements vscode.Disposable {
 			}
 			let simpleTokenCache = new SimpleTokenCache(tokenCacheKey, this._userStoragePath, noSystemKeychain, this._credentialProvider);
 			await simpleTokenCache.init();
+			const cachePath = path.join(this._userStoragePath, './cache.json');
+			let platform = os.platform();
+			let persistence;
```

Comment:
if you're not initializing you should probably provide a type.  Ideally, there would be types in any case, but if it can easily be inferred from the initializer it's probably fine.